plugins/database/src/db_plugin.cpp

```diff
@@ -9536,10 +9575,11 @@ irods::error db_mod_avu_metadata_op(
         addUnits = myUnits;
     }
 
-    status = chlAddAVUMetadata( _ctx.comm(), 0, _type, _name, addAttr, addValue,
-                                addUnits );
-    return CODE( status );
+    // TODO Does this maintain the semantics of admin_mode being set to zero.
+    status = chlAddAVUMetadata( _ctx.comm(), _type, _name, addAttr, addValue,
+                                addUnits, _cond_input );
```

Comment:
remove TODO comment.

if the admin keyword is set, all calls that deal with permissions should honor it.